_ext/team.yaml

```
@@ -310,4 +310,21 @@ team:
     email: alichase@uw.edu
     roles:
       - OHW23 Organizer
+      
+  - name: Myranda Shirk
+    title: Senior Data Scientist
+    affiliate: Vanderbilt University
+    image_url: https://ca.slack-edge.com/TUFFMB3EE-U0291LKH3K4-fe5fb5f59fbd-512
+    github_user: myrandaGoesToSpace
+    # email:
+    roles:
+      - OHW23 Organizer
+      
+  - name: Ben Tupper
+    title: Senior Research Associate
+    affiliation: Bigelow Laboratory for Ocean Sciences
```

The error is probably here. It should be `affiliate` rather than `affiliation`. I'll merge this change.
```suggestion
    affiliate: Bigelow Laboratory for Ocean Sciences
```